code/modules/atmospherics/machinery/trinary/mixer.dm

```diff
@@ -0,0 +1,187 @@
+
+#define _SET_SIGNAL_GAS(GAS, _, _, MIXTURE, ID, ...) signal.data[ID + #GAS] = round(100*##MIXTURE.GAS/MIXTURE##_total_moles);
+#define _RESET_SIGNAL_GAS(GAS, _, _, ID, ...) signal.data[ID + #GAS] = 0;
+#define SET_SIGNAL_MIXTURE(MIXTURE, ID) APPLY_TO_GASES(_SET_SIGNAL_GAS, MIXTURE, ID)
+#define RESET_SIGNAL_MIXTURE(ID) APPLY_TO_GASES(_RESET_SIGNAL_GAS, ID)
+
+/obj/machinery/atmospherics/trinary/mixer
+	name = "Gas mixer"
+	icon = 'icons/obj/atmospherics/mixer.dmi'
+	icon_state = "intact_off"
+	layer = PIPE_MACHINE_LAYER
+	plane = PLANE_NOSHADOW_BELOW
+
+	var/id_tag
+	var/master_id
+	var/on = FALSE
+
+	var/target_pressure = ONE_ATMOSPHERE
+	var/node1_ratio = 0.5
+	var/node2_ratio = 0.5
+
+	var/frequency
+
+/obj/machinery/atmospherics/trinary/mixer/New()
+		..()
+		air3.volume = 300
```

Comment:
why is this not just the default value declared higher up? Does something in a parent override it?

Reply:
The parent usually does 200L, the mixer uses 300L

Reply:
does `air3.volume = 300` not work in the `/obj/machinery/atmospherics/trinary/mixer` part then

Reply:
gotta do it in New, cause otherwise the gas doesn't exist yet.

Reply:
i see